docs/tutorials/modeling.ipynb

```diff
@@ -55,6 +55,7 @@
     "import numpy as np\n",
```

Comment:
<p>I would only mention reoptimize=False in the bullet list, and really conclude the cell with a strong statement on the correct approach for confidence contours.</p><p>It is important to remind that the contours are approximative.  In particular, when the parameter range boundaries are close to the contours. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/gammapy/gammapy/pull/3065/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='44'/>